Update FAQ (#205)
trustwallet/developer: assets/faq.md

Removed contract address will be added to the denylist and, as a result, will no longer be present in token search results inside the TW app.
Why would you want to do this ?.
You are contract owner or just good samaritan who noticed contract to be no longer "active" and was an upgrade and abandoned by owning organization, involved in a scam, mimicking by its name or/and symbol a real contract. All facts must be supported with a link to any resource proving these statements.

> ### Why the price is not displayed on CoinMarketCap?

Reviewer: ```suggestion
### Why token price is not displayed in the app?
```

> Your contract should be displayed on the CoinMarketCap page, if there is no your contract on the page CoinMarketCap we cannot make an association between the token address and CoinMarketCap ID. In this case, contact the team CoinMarketCap and make a request to add an address token.

Reviewer: add link to CMC support page

> Your contract should be displayed on the CoinMarketCap page, if there is no your contract on the page CoinMarketCap (https://support.coinmarketcap.com/hc/en-us/requests/new) we cannot make an association between the token address and CoinMarketCap ID. In this case, contact the team CoinMarketCap and make a request to add an address token.

Reviewer 2: Token prices will only show up within Trust Wallet if the CoinMarketCap (CMC) listing has the correct contract address associated with it. You can submit changes to your CMC listing via their [online form](https://support.coinmarketcap.com/hc/en-us/requests/new). Once an asset's contract address has been added to it's CMC listing, and the asset has enough transaction volume, pricing data will automatically appear in Trust Wallet.

Note that submitting an asset to the Trust Wallet repository does not guarantee can pricing data can or will be associated by CMC.
